lib/matplotlib/axes/_axes.py

```diff
@@ -6220,7 +6225,7 @@ def hist(self, x, bins=None, range=None, normed=False, weights=None,
             if cbook.is_numlike(cumulative) and cumulative < 0:
                 slc = slice(None, None, -1)
 
-            if normed:
+            if density:
```

Comment:
Looks like this path is never tested. Could you add a test that uses the `density` option?